CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.15.0] - 2022-10-11
```

Comment:
Don't we need to update the package json as well?

Reply:
Yes, it's there now.